mlflow/models/evaluation/default_evaluator.py

```diff
@@ -131,22 +130,15 @@ def _infer_model_type_by_labels(labels):
 
 def _extract_raw_model(model):
     model_loader_module = model.metadata.flavors["python_function"]["loader_module"]
+    # If we load a sklearn/xgboost model with mlflow.pyfunc.load_model, the model will be wrapped
+    # with a wrapper, we need to extract the raw model from it.
+    # Extract the raw model of xgboost flavor so that shap explainer uses the raw model
+    # instead of the wrapper and skip data schema validation.
```

Comment:
```suggestion
    # If we load a model with mlflow.pyfunc.load_model, the model will be wrapped
    # with a pyfunc wrapper. We need to extract extract the raw model so that shap
    # explainer uses the raw model instead of the wrapper and skip data schema validation.
```

nit: minor corrections (L133: the wrapper is not only for sklean/xgboost, L135: we extract not only xgboost)